robolectric/src/test/java/org/robolectric/android/internal/ConscryptSecurityModeTest.java

```diff
@@ -0,0 +1,29 @@
+package org.robolectric.android.internal;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.robolectric.annotation.LooperMode.Mode.LEGACY;
+import static org.robolectric.annotation.SecurityMode.Mode.CONSCRYPT;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.LooperMode;
+import org.robolectric.annotation.SecurityMode;
+
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+@RunWith(RobolectricTestRunner.class)
+@LooperMode(LEGACY)
```

Comment:
We might be able to just delete this LooperMode(LEGACY) line. I think it was only really for the BoostrapDelaying runner in the other test.